src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -483,21 +483,33 @@ type WebAppConfig =
                 { site with AppSettings = None; ConnectionStrings = None } // Don't deploy production slot settings as they could cause an app restart
                 for (_,slot) in this.CommonWebConfig.Slots |> Map.toSeq do
                     slot.ToSite site
-                
+            
+            let mutable dependentCustomDomainId = Option<ResourceId>.None
```

Comment:
**Naming**: Dependent is normally something which depnds on, rather than something which is depended on. Perhaps `previousHostnameBinding` would be a better name?

Reply:
Also perhaps a comment explaining why we need these dependencies would be useful?

Reply:
Moved and expanded the existing comment.